### PR TITLE
Tutorials

### PR DESCRIPTION
**Pull request by Stefaanhess**

### CODE REVIEW

**Comment by Stefaanhess** [comment]
Still missing: upload data snippets without labels for model evaluation

File: src/schnetpack/metrics/loss.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
+import torch
+
+
+__all__ = ["mse_loss"]
+
+
+def mse_loss(properties, loss_tradeoff={}):
```
Comment (ktschuett) [inline src/schnetpack/metrics/loss.py]: Call this `build_mse_loss`

File: src/schnetpack/metrics/loss.py [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    """
+    Build the loss function.
```
Comment (ktschuett) [inline src/schnetpack/metrics/loss.py]: mention mean squared error
